unit2-java/lesson17.rst

```diff
@@ -4,14 +4,28 @@ Unit 2 (Java) - Lesson 17 Learning Objectives
 Goals
 -----
 
+- Build persistent model classes that have one-to-many and many-to-many relationships
+- Understand how to model relationships in a relational database
+
 Objectives
 ----------
 
-- Describe the many-to-many relationship and give an example of when this may be a good way to represent data in your database
+One-to-many Relationships
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Describe the one-to-many relationship
+- Use the following annotations and describe what they accomplish:
```

Comment:
This line is rendered bold and italic.  Add an empty line between it and the sub-list.